scripts/build/builders/tizen.py

```diff
@@ -74,13 +74,15 @@ def __init__(self,
             raise Exception("TSAN sanitizer not supported by Tizen toolchain")
 
     def GnBuildArgs(self):
-        if 'TIZEN_HOME' not in os.environ:
+        if 'TIZEN_SDK_ROOT' not in os.environ:
```

Comment:
I would separate code updates from docker updates. Without that, once we check in this code since the docker image we use for development is old, all builds using build_examples.py will break.

Reply:
OK, that makes sense. I will split this PR then.